pkg/scheduler/apis/config/types.go

```diff
@@ -87,6 +87,10 @@ type KubeSchedulerConfiguration struct {
 	// the default value (10s) will be used.
 	PodMaxBackoffSeconds int64
 
+	// IgnoreExistingInterPodAffinities configures the scheduler to ignore existing pods' affinity
+	// rules when scoring candidate nodes, unless the incoming pod has inter-pod affinities.
+	IgnoreExistingInterPodAffinities bool
```

Comment:
Remove this field pls, we only need the one under InterPodAffinityArgs

Reply:
Done